tools/orca/src/kani.rs

```diff
@@ -291,12 +298,11 @@ impl KaniHttpServer {
     }
 
     pub async fn close_connection(&self) {
-        assert!(self
-            .client
+        self.client
             .logout()
             .await
             .map_err(|e| error!("close_connection {:?}", e))
-            .is_ok());
+            .ok();
```

Comment:
.ok() makes this an option, then the `;` causes it to be ignored since close_connection() returns unit. So I think the assert was correct? Were you seeing assertion failures?

Reply:
Yeah it was yelling a lot about 404's and crashing

Reply:
Shouldn't we try and resolve those rather than hiding the error?

Reply:
yeah, I just haven't gotten that far into working out what orca was doing yet 😃 I'll revert that bit for now